### PR TITLE
added instructions to build the book

### LINKED ISSUE #764: images folder missing when creating pdf file
Hello there,
I have cloned the repo and tried to generate a pdf version of the book by using:
`asciidoctor-pdf progit.asc`
I received a lot of error messages informing that all of the *.png images could not be found in directory images:
`image to embed not found or not readable`
It was because directory images does not exist. I had to create one and copy all `*.png` files I could find in the repo:
`lukasz@debkde:~/progit2$ pwd | find -name "*.png" -type f -exec cp {} ./images \;`
After that I could run `asciidoctor-pdf progit.asc` to generate the book. I really appreciate it, thanks.

### PR DESCRIPTION
**Pull request by aollier**
closes #764.

### CODE REVIEW

**Comment by ben** [comment]
I like it. Let me know when you're ready to merge.

**Comment by aollier** [comment]
Now I am ready to merge.
I started the command to copy images with `find` because it is what @bremenpl used whereas it was much simplier. I blame myself for not having thought of using so simply the `cp` command earlier.

**Comment by aollier** [comment]
In fact I am not ready because the next sentence is a proposal:
> You can also use the following commands:

I proposed
> Use the following commands:

Do you agree ?

File: README.asc [hunk 1]
```diff
@@ -27,9 +27,19 @@ Converting to EPub...
 Converting to Mobi (kf8)...
  -- Mobi output at progit.mobi
 Converting to PDF...
- -- PDF  output at progit.pdf
+ -- PDF output at progit.pdf
 ----
 
+An alternative to calling the `bundle` command is to call directly the `asciidoctor` command.
+If it is the first time you build the book, it is necessary to create the _images_ folder and fill it.
```
Comment (ben) [inline README.asc]: "If this is the first time you're building the book…" would be better.

**Comment by ben** [comment]
> I propose
> 
> > Use the following commands:

👍

**Comment by ben** [comment]
:tada:

**Pull request merged**